pkg/oauthflow/flow.go

```diff
@@ -162,15 +160,10 @@ func getCodeFromLocalServer(state string, redirectURL *url.URL) (string, error)
 	}
 }
 
-func randStr() (string, error) {
-	len := 32
-	b := make([]byte, len)
-	n, err := rand.Read(b)
-	if err != nil {
-		return "", err
-	} else if n != len {
-		return "", errors.New("short read")
-	}
+func randStr() string {
```

Comment:
nit: we can just drop the randStr method rather than using it as a stub

Reply:
Yeah, I debated doing that but liked having it there for the comments and rationale for why we're using ksuid. I can be convinced otherwise though :)

Reply:
Ah ok, that's fair.